docs/src/main/tut/apply.md

```diff
@@ -7,7 +7,7 @@ scaladoc: "#cats.Apply"
 ---
 # Apply
 
-`Apply` extends the [`Functor`](functor.html) type class (which features the familiar `map`
+`Apply` extends the [Functor](functor.html) type class (which features the familiar `map`
```

Comment:
I wanted everything to be consistent and the predominant style seemed to be excluding backticks within links. It is also less obvious that the generated doc is a link since the code's grey text-color style takes precedence over the link's blue text-color style.

Given #459 merged, I am happy to revise this PR to make consistent.